hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -65,57 +65,81 @@ class ServiceHootApiDbReaderTest : public HootTestFixture
   CPPUNIT_TEST(runFactoryReadTest);
   CPPUNIT_TEST(runReadWithElemTest);
   CPPUNIT_TEST(runReadByBoundsTest);
+  CPPUNIT_TEST(runAccessPublicMapWithoutEmailTest);
+  CPPUNIT_TEST(runAccessPrivateMapWithoutEmailTest);
+  CPPUNIT_TEST(runInvalidUserTest);
+  // TODO: fix
+//  CPPUNIT_TEST(runMultipleMapsSameNameDifferentUsersPrivateTest);
+//  CPPUNIT_TEST(runMultipleMapsSameNameDifferentUsersPublicTest);
+//  CPPUNIT_TEST(runMultipleMapsSameNameNoUserPublicTest);
   CPPUNIT_TEST_SUITE_END();
 
 public:
 
-  QString userEmail() { return QString("%1.ServiceHootApiDbReaderTest@hoottestcpp.org").arg(testName); }
-
-  long mapId;
-  QString testName;
+  QString userEmail()
+  { return QString("%1.ServiceHootApiDbReaderTest@hoottestcpp.org").arg(_testName); }
 
   ServiceHootApiDbReaderTest()
   {
     setResetType(ResetAll);
     TestUtils::mkpath("test-output/io/ServiceHootApiDbReaderTest");
   }
 
-  void setUpTest(const QString& test_name)
+  void setUpTest(const QString testName)
   {
-    mapId = -1;
-    testName = test_name;
+    _mapId = -1;
+    _testName = testName;
     ServicesDbTestUtils::deleteUser(userEmail());
     HootApiDb database;
 
     database.open(ServicesDbTestUtils::getDbModifyUrl());
-    database.getOrCreateUser(userEmail(), QString("%1.ServiceHootApiDbReaderTest").arg(testName));
+    database.getOrCreateUser(userEmail(), QString("%1.ServiceHootApiDbReaderTest").arg(_testName));
     database.close();
   }
 
   virtual void tearDown()
   {
     ServicesDbTestUtils::deleteUser(userEmail());
 
-    if (mapId != -1)
+    if (_mapId != -1)
     {
       HootApiDb database;
       database.open(ServicesDbTestUtils::getDbModifyUrl());
-      database.deleteMap(mapId);
+
+      database.deleteFolderMapMappingsByMapId(_mapId);
+      database.deleteFolders(database.getFolderIdsAssociatedWithMap(_mapId));
```

Comment:
Good point.  The delete code is only used for testing purposes, so I'll move it over to the test utilities and add a warning to the comments that no additional checking occurs.  Every test generates a unique map ID, so it will work for that purpose.